liquibase-core/src/main/resources/www.liquibase.org/xml/ns/dbchangelog/dbchangelog-latest.xsd

```diff
@@ -1004,6 +1004,7 @@
             <xsd:attribute name="tablespace" type="xsd:string"/>
             <xsd:attribute name="tableType" type="xsd:string"/>
             <xsd:attribute name="remarks" type="xsd:string"/>
+            <xsd:attribute name="enableIfNotExistsStatement" type="xsd:boolean"/>
```

Comment:
I believe the attribute name could be shorter, like "ifNotExists" ?

Reply:
@filipelautert fixed it in ec46d1dc271163ea395cfb0a60ed6f1822f7757e